spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/AcknowledgeablePubsubMessage.java

```diff
@@ -38,9 +39,26 @@
 	 */
 	String getSubscriptionName();
 
+	/**
+	 * Acknowledge the message asynchronously
+	 */
+	void ackAsync();
```

Comment:
I think we should return `java.util.concurrent.Future` for all of these async methods.